app/server/api/disbursement_api.py

```diff
@@ -77,7 +79,6 @@ def make_transfers(disbursement_id, auto_resolve=False):
         yield {
             'message': 'success' if percent_complete == 100 else 'pending',
             'percent_complete': math.floor(percent_complete),
-            'data': {'credit_transfers': credit_transfers_schema.dump(disbursement.credit_transfers).data}
```

Comment:
This is just read by the (as yet non-existent) progress bar. This data isn't needed there at all, and it's big and expensive. Deleting this was a big win.